ocs_sample_library_preview/BaseClient.py

```diff
@@ -114,6 +114,16 @@ def communityHeaders(self, community_id: str):
 
         return headers
 
+    def dataViewNonVerboseHeader(self):
```

Comment:
Should this be more generically be called ```nonVerboseSdsHeaders```? I could see the benefit for other API calls besides just dataViews in the future. Another solution I could envision would be to just add optional override parameters in the sdsHeaders function.

Reply:
good catch, there's nothing data view specific about this function, aside from data views being the only one that should be calling it. Separately, I do think we can refactor the headers section entirely so that we can specify additional headers easily in line, but that's not too trivial. I'll make a work item for it actually

Reply:
changed